metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java

```diff
@@ -263,8 +263,7 @@ private void validateTableType(final QualifiedName tableName, final Map<String,
             if (ICEBERG_TABLE_TYPE.equalsIgnoreCase(tableMetadata.get(PARAM_TABLE_TYPE))) {
                 return;
             }
-            if (VIRTUAL_VIEW_TABLE_TYPE.equalsIgnoreCase(tableMetadata.get(PARAM_TABLE_TYPE))
-                && tableMetadata.containsKey(COMMON_VIEW)
+            if (tableMetadata.containsKey(COMMON_VIEW)
```

Comment:
The two condition check can be replaced with one. Boolean.parseBoolean(tableMetadata.get(COMMON_VIEW))

Reply:
Changed based on comments with a bit code refactor.